src/components/TableCard.tsx

```diff
@@ -0,0 +1,102 @@
+import { ReactNode, useMemo } from "react";
+import { useHover } from "react-aria";
+import { Css, Palette } from "src/Css";
+import { ButtonMenu, MenuItem } from "./ButtonMenu";
+import { Tag, TagType } from "./Tag";
+
+export type TableCardType = "card" | "list";
+
+type CardTag = {
+  text: string;
+  type?: TagType;
+};
+
+export interface TableCardProps {
+  title: string;
+  subtitle: string;
+  metadata: ReactNode;
```

Comment:
nit: Maybe just call this `content` since it's just a block to put whatever details you want?

Reply:
It seems like I can make this optional, but I'll double check.

`content` makes me think it's a more encompassing part of the component instead of just a small extra details section. What do you think about calling it something that has more of an indication that it just adds some extra information, like `details`, `detailContent`, `additionalContent`? idk...